frontend/packages/console-app/src/components/pdb/PDBList.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { sortable } from '@patternfly/react-table';
+import * as classNames from 'classnames';
+import i18next from 'i18next';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { RowProps, TableColumn } from '@console/dynamic-plugin-sdk';
+import { useActiveColumns } from '@console/internal/components/factory/Table/active-columns-hook';
+import VirtualizedTable, {
+  TableData,
+} from '@console/internal/components/factory/Table/VirtualizedTable';
+import {
+  Kebab,
+  ResourceLink,
+  Timestamp,
+  ResourceKebab,
+  Selector,
+} from '@console/internal/components/utils';
+import { PodDisruptionBudgetModel } from '@console/internal/models';
+import { PodDisruptionBudgetKind, referenceForModel } from '@console/internal/module/k8s';
+
+const tableColumnInfo = [
+  { className: '', id: 'name' },
+  { className: '', id: 'namespace' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-sm'), id: 'selector' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-md'), id: 'minAvailable' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-lg'), id: 'disruptionsAllowed' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-xl'), id: 'creationTimestamp' },
+  { className: Kebab.columnClass, id: '' },
+];
+
+const { common } = Kebab.factory;
+const menuActions = [...Kebab.getExtensionsActionsForKind(PodDisruptionBudgetModel), ...common];
+
+const PodDisruptionBudgetTableRow: React.FC<RowProps<PodDisruptionBudgetKind>> = ({
+  obj,
+  activeColumnIDs,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <TableData {...tableColumnInfo[0]} activeColumnIDs={activeColumnIDs}>
+        <ResourceLink
+          kind={referenceForModel(PodDisruptionBudgetModel)}
+          name={obj.metadata.name}
+          namespace={obj.metadata.namespace}
+        />
+      </TableData>
+      <TableData {...tableColumnInfo[1]} activeColumnIDs={activeColumnIDs}>
+        <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
+      </TableData>
+      <TableData {...tableColumnInfo[2]} activeColumnIDs={activeColumnIDs}>
+        <Selector selector={obj.spec.selector} namespace={obj.metadata.namespace} />
+      </TableData>
+      <TableData {...tableColumnInfo[3]} activeColumnIDs={activeColumnIDs}>
+        {_.isNil(obj.spec.maxUnavailable)
+          ? `${t('console-app~Min available')} ${obj.spec.minAvailable}`
+          : `${t('console-app~Max unavailable')} ${obj.spec.maxUnavailable}`}
```

Comment:
If you have neither, you get `undefined` in the UI... 

![image](https://user-images.githubusercontent.com/8126518/162489559-3e1fd65e-6312-4f84-bd9d-5f01cd802722.png)

We probably want to check with UX... but I'd think `Not set` or something would be more appropriate.

Reply:
> 

@andrewballantyne This happens when creating pdb from editor. The 'minAvailable' and `maxUnavailable` are required field on the form view. I have seen `-` used in console for undefined property.

Reply:
Yeah, we use DASH when we are trying to do specific empty field stuff in the rows of tables.

If the YAML editor does not fail on submit (which should be as close to CLI as the UI will get), we need to support it. We can make the form have more guidance for the user, but the rest of the code needs to support the lowest common denominator.

Reply:
> Yeah, we use DASH when we are trying to do specific empty field stuff in the rows of tables.
> 
> If the YAML editor does not fail on submit (which should be as close to CLI as the UI will get), we need to support it. We can make the form have more guidance for the user, but the rest of the code needs to support the lowest common denominator.

@andrewballantyne The availability and selector  are required fields  in the design docs. Will remove the required validation for them,  and communicate the same to UX team. @megan-hall @thile01